lib/ui/forgot_password/forgot_password_screen.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:survey_flutter_ic/gen/assets.gen.dart';
+import 'package:survey_flutter_ic/ui/widget/input_field_widget.dart';
+import 'package:survey_flutter_ic/utils/dimension.dart';
+
+class ForgotPasswordScreen extends StatelessWidget {
+  const ForgotPasswordScreen({
+    Key? key,
+  }) : super(key: key);
```

Comment:
I think we can shorten it to be like this

```suggestion
  const ForgotPasswordScreen({super.key});
```